src/css/style.css

```diff
@@ -11,7 +11,7 @@ body {
 }
 .gpii-fd,
 .gpii-fd p {
-    font-size: 2.5rem; /*base font size */
+    font-size: 2.25rem; /*base font size */
```

Comment:
I think this has changed how the indicator bar for the text size is displayed. It seems off centre now.
![screen shot 2015-04-23 at 9 09 25 am](https://cloud.githubusercontent.com/assets/556024/7297703/741b23d0-e998-11e4-9afc-ddb58d579c0f.png)